### PR TITLE
Add demo file for UI testing

### PR DESCRIPTION
**Pull request by tjcolvin**

### CODE REVIEW

**Comment by schrags** [comment]
@tjcolvin this is amazing

File: tjcolvin-demo-file.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Test a new file
+
+Header 1 | Header 2
```
Comment (tjcolvin) [inline tjcolvin-demo-file.md:3]: @schrags Interesting feature. Have you tried it yet? :bangbang:

File: tjcolvin-demo-file.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+---------|---------
+test 1.1 | test 2.1
```
Comment (schrags) [inline tjcolvin-demo-file.md:5]: test

Comment (tjcolvin) [inline tjcolvin-demo-file.md:5]: a test for line notes

**Comment by tjcolvin** [comment]
:+1:  :ship: :shipit:

**Pull request merged**